typescript/sdk/src/middleware/deploy.ts

```diff
@@ -14,25 +14,34 @@ import { ProxiedContract, ProxyKind } from '../proxy';
 import { HyperlaneRouterDeployer } from '../router/HyperlaneRouterDeployer';
 import {
   ProxiedRouterContracts,
+  ProxiedRouterFactories,
   RouterConfig,
-  RouterFactories,
 } from '../router/types';
 import { ChainMap, ChainName } from '../types';
 
 export type InterchainAccountFactories =
-  RouterFactories<InterchainAccountRouter>;
+  ProxiedRouterFactories<InterchainAccountRouter>;
 
 export const interchainAccountFactories: InterchainAccountFactories = {
   router: new InterchainAccountRouter__factory(),
+  // TODO: where to put these?
```

Comment:
What are these used for? Please add a comment explaining the presence of this and "router"

Reply:
removed

Reply:
removed these
was a hack to help with the serialization transform but deciding to punt